capstone/src/main/java/com/step902020/capstone/OrganizationController.java

```diff
@@ -92,7 +95,25 @@ public RedirectView saveOrganization(
    * @return Organization object
    */
   @GetMapping("get-public-profile")
-  public Organization getPublicProfile(@RequestParam("organization-id") String organizationId) {
+  public Organization getPublicProfile(@RequestParam("organization-id") String organizationId) throws IOException {
     return this.organizationRepository.findById(Long.parseLong(organizationId)).orElse(null);
   }
+
+  /**
+   * deletes event from the organization's event list and deletes the event entity
+   * @param eventId datastore id of the event to be deleted
+   * @param user current user
+   * @return RedirectView to manage event page
+   * @throws IOException
+   */
+  @PostMapping("delete-organization-event")
+  public RedirectView deleteOrganizationEvent(@RequestParam("event-id") String eventId, CurrentUser user) throws IOException {
+    Organization current = this.organizationRepository.findByEmail(user.getEmail()).get(0);
```

Comment:
By the way, instead of getting the 0th element manually, you can change the `findByEmail()` method to instead be `findFirstByEmail()`.
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.limit-query-result

Reply:
Fixed in the other branch